kalite/distributed/static/js/distributed/exercises/perseus-helpers.js

```diff
@@ -99,25 +112,17 @@ Exercises.PerseusBridge = {
     render_item: function(item_data) {
 
         Exercises.PerseusBridge.itemMountNode = document.createElement("div");
-        var itemRenderer = Exercises.PerseusBridge.itemRenderer = Perseus.ItemRenderer({
+
+        var ItemRenderer = React.createFactory(Perseus.ItemRenderer);
+        Exercises.PerseusBridge.itemRenderer = zk = React.render(ItemRenderer({
             item: item_data,
             problemNum: Math.floor(Math.random() * 50) + 1,
```

Comment:
I think we just took this directly from the Khan Academy example html file inside the Perseus repo. Will take a https://twitter.com/stealthmountain at what they do on the live site.